src/server/scripts/Spells/spell_monk.cpp

```diff
@@ -30,10 +30,10 @@ enum MonkSpells
 {
     SPELL_MONK_CRACKLING_JADE_LIGHTNING_CHANNEL = 117952,
     SPELL_MONK_CRACKLING_JADE_LIGHTNING_CHI_PROC = 123333,
-    SPELL_MONK_STANCE_OF_THE_SPIRITED_CRANE = 154436,
-
     SPELL_MONK_CRACKLING_JADE_LIGHTNING_KNOCKBACK = 117962,
     SPELL_MONK_CRACKLING_JADE_LIGHTNING_KNOCKBACK_CD = 117953,
+    SPELL_MONK_PROVOKE = 118635,
```

Comment:
http://www.wowhead.com/spell=116189/provoke
http://www.wowhead.com/spell=118635/provoke

your spell is the second, which is a area taunt... this is wrong and I think it should be casted  by the statue if I remember correctly.